src/main/java/bio/terra/cli/businessobject/Context.java

```diff
@@ -117,22 +118,28 @@ public static void synchronizeToDisk() {
    */
   public static Path getContextDir() {
     // default to the user's home directory
-    Path parentDir = Paths.get(System.getProperty("user.home"));
-
-    // if the override environment variable is set and points to a valid directory, then use it
-    // instead
+    String contextDir = System.getProperty("user.home");
+    // if the override environment variable is set, use it instead
     String overrideDirName = System.getenv(CONTEXT_DIR_OVERRIDE_NAME);
     if (overrideDirName != null && !overrideDirName.isBlank()) {
-      Path overrideDir = Paths.get(overrideDirName).toAbsolutePath();
-      if (overrideDir.toFile().exists() && overrideDir.toFile().isDirectory()) {
-        parentDir = overrideDir;
-      } else {
-        throw new UserActionableException(
-            "Override environment variable does not point to a valid directory: " + overrideDir);
-      }
+      contextDir = overrideDirName;
+    }
+    // If this is a test, append the current runner's ID. This lets us run multiple tests in
+    // parallel without clobbering context across runners.
+    String isTest = System.getProperty(CommandRunner.IS_TEST);
+    if (isTest != null && isTest.equals("true")) {
+      contextDir += System.getProperty("org.gradle.test.worker");
+    }
+
+    Path contextPath = Paths.get(contextDir).toAbsolutePath();
+    // build.gradle test task makes contextDir. However, with Gradle Enterprise Test Distribution,
```

Comment:
opt - revert 134-139 since it was needed for TD. Also put `mkdir` back in build.gradle

Reply:
I think we now need this to create the runner-specific dirs. I'm not sure if we can tell how many threads will actually be used from `build.gradle`, since `maxParallelForks` is just the limit